code/__HELPERS/see_through_maps.dm

```diff
@@ -0,0 +1,30 @@
+//global statics for the see_through_component coordinate maps
+//For ease of use, include a comment in the shape of the coordinate map, where O is nothing, X is a hidden tile and A is the object
+GLOBAL_LIST_INIT(see_through_maps, list(
+	// X
+	// A
+	"default" = list(
+						list(0, 1, 0)
```

Comment:
only indent once

Reply:
It wont line up anymore with the relative coordinates then though
![image](https://user-images.githubusercontent.com/7501474/188086675-52e9f186-c22b-46a6-aa65-1ec3610e8848.png)

Reply:
it already doesn't because people have separate tab widths. if you really want to try to align here (and i would say this is a fine case), do something like `/* xxxxxx */` with the appropriate number of x's

Reply:
I'll do that then, thank you for the explanation!